javascript/scripts.js

```diff
@@ -0,0 +1,32 @@
+let playerOne = document.querySelector('.player-one');
+let playerTwo = document.querySelector('.player-two');
+let playerThree = document.querySelector('.player-three');
+let game = new Game();
+
+
+
+$('.start-btn').on('click', () => {
+  createPlayers();
+  game.start();
+})
+
+createPlayers = () => {
```

Comment:
I see that you started creating a `createPlayers` method on your Game class --- which is where that functionality should exist. I'd get rid of this function in this file all together and in your click handler for `.start-btn`, call `game.start()` and `game.createPlayers()` (perhaps those two methods could even just be combined into a single method called `game.init()`